### PR TITLE
[300] Joint Rupture Experiment Write-up

### PR DESCRIPTION
**Pull request by voj**
The is the write-up for #300 with a description of the code, the results, and future work.

### CODE REVIEW

File: doc/investigations/300-joint-ruptures/readme.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Driver class is in [JointRupturePostProcessor](..%2F..%2F..%2Fsrc%2Fmain%2Fjava%2Fnz%2Fcri%2Fgns%2FNZSHM22%2Fopensha%2Fruptures%2Fexperimental%2Fjoint%2FJointRupturePostProcessor.java)
+
+Heavy lifting happens in [JointRuptureBuilderParallel](..%2F..%2F..%2Fsrc%2Fmain%2Fjava%2Fnz%2Fcri%2Fgns%2FNZSHM22%2Fopensha%2Fruptures%2Fexperimental%2FJointRuptureBuilderParallel.java)
+
+Similar approach to building single-stranded ruptures, but here we combine a subduction and a crustal rupture if a minimum number of sections are close enough. More than one crustal rupture can be linked to a subduction rupture, resulting in more than one splay.
+
+Any other physical features such as rake, dip, relative azimuth are not yet taken into account.
+
+## Use Existing Coulomb Filters to Create Joint Ruptures
```
Comment (chrisdicaprio) [inline doc/investigations/300-joint-ruptures/readme.md:59]: would it be worth a brief summary of how the "standard" Coulomb filter works? That would help put the new Coulomb filter into perspective

File: doc/investigations/300-joint-ruptures/readme.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Considerations:
+- Thinning should not change the size or magnitude distribution of ruptures
+- Thinning should not bias one region against another. Thinning needs to be applied evenly over NZ.
+
+Explored strategies:
+
+- Subduction filtering:
+    - size increase needs to be at least 10%
+    - ruptures of the same size avoid overlapping
```
Comment (chrisdicaprio) [inline doc/investigations/300-joint-ruptures/readme.md:116]: not sure I understand what this means

File: doc/investigations/300-joint-ruptures/readme.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+- Crustal ruptures:
```
Comment (chrisdicaprio) [inline doc/investigations/300-joint-ruptures/readme.md:117]: 10 feels very restrictive. Can we open the aperture and still have a manageable number of possible ruptures?

File: doc/investigations/300-joint-ruptures/readme.md (hunk 1 above)
Comment (voj) [reply doc/investigations/300-joint-ruptures/readme.md:59]: done

File: doc/investigations/300-joint-ruptures/readme.md (hunk 2 above)
Comment (voj) [reply doc/investigations/300-joint-ruptures/readme.md:116]: done

File: doc/investigations/300-joint-ruptures/readme.md (hunk 3 above)
Comment (voj) [reply doc/investigations/300-joint-ruptures/readme.md:117]: done

Comment (voj) [reply doc/investigations/300-joint-ruptures/readme.md:119]: (explained that I haven't explored this yet)

**Review by chrisbc — APPROVED** [review]
Nice - thanks @voj

**Pull request merged**